### DISCUSSION TITLE
Error when sending confirmation e-mails

### DISCUSSION THREAD

**Discussion by elpaiton** [pocketbase/pocketbase#591] [Q&A]
Hello everyone, I hope you are very well.

I have hung my pocketbase on an AWS instance, everything seems to be fine, but when I try to send confirmation and verification emails I get an error. 

I get the following error:

Failed to send the test email. Raw error: 
535 Authentication Failed.

Would anyone know what is going on? Thank you for your response.

**Comment by a10d** [comment]
The error message [`535 Authentication Failed`](https://en.wikipedia.org/wiki/List_of_SMTP_server_return_codes#%E2%80%94_5yz_Permanent_negative_completion) comes from the SMTP server you're using and means your credentials are invalid.
Check your server settings :)

**Comment by ganigeorgiev** [comment]
@Jasics0 As the others mentioned, make sure to check that your credentials are valid and you are using the correct port (for TLS/STARTTLS that's usually 587).

_I'll move the issue to Q&A discussion._
